### ISSUE TITLE
[Enhancement]: Support reporting the total size of writable space in the cluster.

### ISSUE THREAD

**Issue by Victor1319** [cubefs/cubefs#2658]
### Is there an existing issue for this?

- [X] I have searched all the existing issues

### What would you like to be added?

Support only calculating the total size of writable space in the cluster.

### Why is this needed?

The current total size includes the size of non-writable nodes, making it difficult to accurately assess the available size of the cluster

**Comment by NaturalSelect** [comment]
This issue is work in progress?

**Comment by awaqwqa** [comment]
我要报名这个

**Comment by morphes1995** [comment]
i want to try this !

**Comment by morphes1995** [comment]
In case of all nodes are online, cluster info shows as follows. In order to represent capacity more accurately, i use float number

```
root@ubuntu01:~/# cfs-cli cluster info                                                
[Cluster]                                                                                                                                                                                
  Cluster name       : cfs_cluster          
  Master leader      : 10.24.20.227:17010                                                                                                                                                
  Master-3           : 10.24.20.229:17010
  Master-1           : 10.24.20.227:17010                                                   
  Master-2           : 10.24.20.228:17010
  Auto allocate      : Enabled                                                              
  MetaNode count (active/total)    : 3/3      
  MetaNode used                    : 0.284 GB                                               
  MetaNode Available               : 7.754 GB
  MetaNode total                   : 8.038 GB                                                                                                                                            
  DataNode count (active/total)    : 3/3     
  DataNode used                    : 71.998 GB                                              
  DataNode Available               : 185.955 GB                                             
  DataNode total                   : 272.698 GB                                                                                                                                          
  Volume count                     : 2                                                                                                                                                   
  Allow Mp Decomm                  : Enabled                                                                                                                                             
  EbsAddr                          :                                                        
  LoadFactor                       : 0                                                                                                                                                   
  BatchCount         : 0                
  MarkDeleteRate     : 0                     
  DeleteWorkerSleepMs: 0                                                                                                                                                                 
  AutoRepairRate     : 0                                                                    
  MaxDpCntLimit      : 3000
```
After a node down and wait couple of seconds,  active metanode/datanode count and metanode/datanode available space changed 

```
root@ubuntu01:~# cfs-cli cluster info 
[Cluster]
  Cluster name       : cfs_cluster
  Master leader      : 10.24.20.227:17010
  Master-3           : 10.24.20.229:17010
  Master-1           : 10.24.20.227:17010
  Master-2           : 10.24.20.228:17010
  Auto allocate      : Enabled
  MetaNode count (active/total)    : 2/3
  MetaNode used                    : 0.285 GB
  MetaNode Available               : 5.178 GB
  MetaNode total                   : 8.038 GB
  DataNode count (active/total)    : 2/3
  DataNode used                    : 72.011 GB
  DataNode Available               : 117.625 GB
  DataNode total                   : 272.698 GB
  Volume count                     : 2
  Allow Mp Decomm                  : Enabled
  EbsAddr                          : 
  LoadFactor                       : 0
  BatchCount         : 0
  MarkDeleteRate     : 0
  DeleteWorkerSleepMs: 0
  AutoRepairRate     : 0
  MaxDpCntLimit      : 3000

```
my pr is https://github.com/cubefs/cubefs/pull/3017
